components/header/README.md

```diff
@@ -7,8 +7,61 @@ Header
 ```
 <!-- STORY -->
 
+### Usage
 
 
+Simple
+```js
+ <div>
+   <Header level={1}>A 48px Bold heading</Header>
+   <Header level={2}>A 36px Bold heading</Header>
+   <Header level={3}>A 24px Bold heading</Header>
+   <Header level={4}>A 19px Bold heading</Header>
+   <Header level={5}>h5</Header>
+   <Header level={6}>h6</Header>
+ </div>
+```
+
+Shortcuts
+```js
+import { H1, H2, H3, H4, H5, H6 } from "@govuk-react/header";
```

Comment:
Listing all the examples here could be good tho, to showcase the shortcuts available, maybe just remove the anonymous div on this example?